app/models/concerns/person.rb

```diff
@@ -13,7 +13,9 @@ module Person
     validates :date_of_birth,
       presence: true,
       inclusion: {
-        in: ->(_) { MAX_AGE.years.ago.beginning_of_year.to_date..Time.zone.today }
+        in: lambda do |_|
```

Comment:
When it comes to lambdas I think we should be consistent and stick with one syntax style.